shoes-swt/spec/shoes/swt/text_block/painter_spec.rb

```diff
@@ -13,7 +13,7 @@
   let(:graphics_context) do
     double("graphics context", set_antialias: nil, set_line_cap: nil,
                                set_transform: nil, clipping: nil,
-                               set_clipping: nil)
+                               set_clipping: nil, set_alpha: nil)
```

Comment:
Good point. Will get that covered for real somewhere, as these really just stub unnecessary stuff for painters to run.